homeassistant/components/group/__init__.py

```diff
@@ -341,6 +342,37 @@ async def _async_process_config(hass, config, component):
         )
 
 
+class GroupEntity(Entity):
+    """Representation of a Group of entities."""
+
+    @property
+    def should_poll(self) -> bool:
+        """Disable polling for group."""
+        return False
+
+    async def async_added_to_hass(self):
+        """Register listeners."""
+
+        @callback
+        def _update_at_start(_):
+            self.async_schedule_update_ha_state(True)
+
+        self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _update_at_start)
+
+    @callback
+    def async_schedule_or_defer_update_ha_state(
+        self, force_refresh: bool = False
+    ) -> None:
+        """Only update once at start."""
+        if not self.hass:
```

Comment:
We should not need this. If someone calls this before the Entity is added to hass, we need fix that bug.

Reply:
It was just to make mypy happy.  I'll change it to an assert.